### PR TITLE
feat: created login and signup pages

### PR DESCRIPTION
**Pull request by nakulsharma15**
Feat: Created Login and Signup Pages

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
The code clean and good simple login and signup form.

File: src/Pages/Login.jsx [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+                </div>
+                <div className="flex-column">
+                    <label for="password">Password:</label>
+                    <input className="login-input" type="password" id="password" required/>
+                </div>
+                <div>
+                    <input type="checkbox" id="remember" />
+                    <label for="confirm">Remember me</label>
+
+                    <span className="signup">Don't have an account?, <Link classNameName="signup-link" to="/signup">Signup</Link></span>
```
Comment (coldpigli) [inline src/Pages/Login.jsx]: in link it should be className instead of classNameName. Fix typo

**Review by coldpigli — APPROVED** [review]
Found a typo. Everything else looks good.

File: src/Pages/Login.jsx (hunk 1 above)
Comment (nakulsharma15) [reply src/Pages/Login.jsx]: Thanks for pointing out. Fixed it in the latest commit.

**Pull request merged**